web/packages/core/src/plugin-polyfill.ts

```diff
@@ -216,6 +228,9 @@ declare global {
  * @param plugin The plugin to install
  */
 export function installPlugin(plugin: RufflePlugin): void {
+    if (isFlashEnabledBrowser()) {
```

Comment:
For symmetricity, can this be moved into `pluginPolyfill()` in `polyfills.ts`? Then `isFlashEnabledBrowser` can be moved there as well without an `export`.

Reply:
It can, though that function is not called by the Ruffle extension, which calls installPlugin directly, but if you explicitly install the Ruffle extension on a Flash-enabled browser, you probably want to use Ruffle. In fact, perhaps, I should also change `if (!isFlashEnabledBrowser() && (isExtension || !usingExtension)) {` to `if (isExtension || (!isFlashEnabledBrowser() && !usingExtension)) {`